flexmeasures/ui/templates/views/sensors.html

```diff
@@ -105,7 +105,14 @@
           .then(function(data) {
             if (data.timezone != jstz.determine().name()) {
               document.getElementById('tzwarn').style.display = 'block';
-              document.getElementById('tzwarn').innerHTML = 'Please note that the sensor data you are viewing is located in a different timezone.<br/>To view the data from a local perspective, set your locale timezone to ' + data.timezone + '.'
+              var offsetDifference = getOffsetBetweenTimezonesForDate(new Date(), data.timezone, jstz.determine().name());
+              if (offsetDifference != 0) {
+                var offsetNotice = (offsetDifference > 0) ? 'which is currently ahead by ' + offsetDifference + 'minutes' : 'which is currently behind by ' + offsetDifference + ' minutes'
+                document.getElementById('tzwarn').innerHTML = 'Please note that the sensor data you are viewing is located in a different timezone (' + offsetNotice + ').<br/>To view the data from a local perspective, set your locale timezone to ' + data.timezone + '.'
+              }
+              else {
+                document.getElementById('tzwarn').innerHTML = 'Please note that the sensor data you are viewing is located in another timezone (' + data.timezone + '), but this seems to behave similar to yours (' + jstz.determine().name() + ').'
```

Comment:
Why would this be necessary if the timezone behaves "similar"?

Reply:
We're only checking whether the offsets of both timezones are the same _now_ (time of page load). However, it is possible for two timezones to have the same offset only for a few hours or a few days until they diverge again. I didn't want to go overboard with this logic.

As an alternative, we could choose to not show the warning at all (in case both timezones share their current offset), and accept the risk of not reporting a potential difference in those arguably rare cases where:
- the user and asset are in a different timezone,
- both timezones share an offset _now_ (time of page load), and
- the timezones actually do have a different offset for the time window being viewed
I can only imagine DST transitions at various times or dates to be the root cause of such cases, and then the difference would be 1 hour at most anyways.